giskard/scanner/robustness/text_transformations.py

```diff
@@ -122,6 +122,57 @@ def _random_key_typo(self, char):
             return typo if char.islower() else typo.upper()
         return char
 
+class TextFromSpeechTypoTransformation(TextTransformation):
+    name = "Add text from speech typos"
+    
+    def __init__(self, column, rate=0.05, min_length=10, rng_seed=1729):
+        super().__init__(column)
+        from .entity_swap import speech_typos
+
+        self.rate = rate
+        self.min_length = min_length
+        self._word_typos = speech_typos
+        self.rng = np.random.default_rng(seed=rng_seed)
+
+    def make_perturbation(self, x):
+        # Skip if the text is too short
+        if len(x) < self.min_length:
+            return x
+
```

Comment:
I guess we would want to skip for languages other than English too

```suggestion
        # Skip if language isn't supported
        if row["language__gsk__meta"] not 'en':
            return x 
```

Reply:
Just fixed compilation